src/EditLabelDialog/EditLabelDialog.jsx

```diff
@@ -24,7 +24,7 @@ export default function EditLabelDialog({
   onNew = () => {},
   onEdit = () => {},
   onClose = () => {},
-  labelDialogTitle = "Edit Label",
+  labelDialogTitle = "Add Label",
```

Comment:
A better property name would be `title`. The component name already contains Label and Dialog.